deploy/csi-azuredisk-controller.yaml

```diff
@@ -169,7 +170,7 @@ spec:
               drop:
                 - ALL
         - name: azuredisk
-          image: mcr.microsoft.com/k8s/csi/azuredisk-csi:latest
```

Comment:
remove this change since we always uses `mcr.microsoft.com/k8s/csi/azuredisk-csi:latest` for latest image